packages/atomic/src/components/search/facets/facet-segmented-value/facet-segmented-value.ts

```diff
@@ -0,0 +1,70 @@
+import {html} from 'lit';
+import {ifDefined} from 'lit/directives/if-defined.js';
+import {keyed} from 'lit/directives/keyed.js';
+import {renderButton} from '@/src/components/common/button';
+import type {FacetValuePropsBase} from '@/src/components/common/facets/facet-common';
+import type {FunctionalComponent} from '@/src/utils/functional-component-utils';
+
+export const renderFacetSegmentedValue: FunctionalComponent<
+  FacetValuePropsBase
+> = ({props}) => {
+  const compactCount = new Intl.NumberFormat(props.i18n.language, {
+    notation: 'compact',
+  }).format(props.numberOfResults);
+
+  const count = props.numberOfResults.toLocaleString(props.i18n.language);
+
+  const ariaLabel = props.i18n.t('facet-value', {
+    value: props.displayValue,
+    count: props.numberOfResults,
+    formattedCount: compactCount,
+  });
+
+  return html`
+    ${keyed(
+      props.displayValue,
+      html`<li>
+        ${renderButton({
+          props: {
+            style: 'square-neutral',
+            part: `value-box${props.isSelected ? ' value-box-selected' : ''}`,
+            onClick: () => props.onClick(),
+            class: `value-box group box-border flex h-full items-center p-2 ${
+              props.isSelected
+                ? 'selected border-primary shadow-inner-primary'
+                : 'hover:border-primary-light focus-visible:border-primary-light'
+            }`,
+            ariaPressed: props.isSelected ? 'true' : 'false',
+            ariaLabel,
+            ref: props.buttonRef,
+          },
+        })(
+          html`<span
+              title=${ifDefined(props.displayValue)}
+              part="value-label"
+              class="value-label truncate ${
+                props.isSelected
+                  ? 'text-primary'
+                  : 'group-hover:text-primary-light group-focus:text-primary'
+              }"
+            >
+              ${props.displayValue}
+            </span>
+            <span
+              title=${ifDefined(count)}
```

Comment:
Nit: `count` should always be defined, so again the `ifDefined` here is superfluous.